manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "0Auth Extension",
+  "version": "1.0",
+  "description": "Chrome Extension for 0Auth",
+  "permissions": ["declarativeContent", "storage"],
+  "browser_action": {
+    "default_popup": "build/index.html",
+    "default_title": "0Auth",
+    "default_icon": {
+      "128": "public/icon.png"
+    }
+  },
+  "icons": {
+    "128": "public/icon.png"
+  },
+  "content_security_policy": "script-src 'self' 'sha256-qILRdnleFFB8nUofQbeggst+MRykHHDGiqtfJvKdtPY='",
```

Comment:
Each time we build react, we have to enter a new hash value.
This is a value to confirm that the code of the extension has not been changed.